test/debugger/test-debugger-remote.js

```diff
@@ -28,24 +28,20 @@ interfacer.stdout.on('data', function(data) {
   });
 });
 
-interfacer.on('line', function(line) {
+interfacer.on('line', common.mustCall(function(line) {
   line = line.replace(/^(debug> *)+/, '');
-  console.log(line);
-  var expected = '\bconnecting to localhost:5959 ... ok';
-  assert.ok(expected == line, 'Got unexpected line: ' + line);
-});
+  const expected = '\bconnecting to localhost:5959 ... ok';
+  assert.strictEqual(expected, line, 'Got unexpected line: ' + line);
```

Comment:
nit: `expected` should be the second argument. Also I think the message can be dropped now